tmol/database/default/scoring/ref.yaml

```diff
@@ -0,0 +1,21 @@
+weights: 
+  ALA : 2.3386
+  ARG : 3.2718
+  ASN : -2.2837
+  ASP : -2.5358
+  CYS : 1.4028
+  GLN : 1.2108
+  GLU : 0.134426
+  GLY : 1.0317
+  HIS : -1.6738
+  ILE : 0.729516
+  LEU : 1.2334
+  LYS : -0.873554
+  MET : -5.1227
+  PHE : -1.0644
+  PRO : -1.281
+  SER : -1.1772
+  THR : -1.425
+  TRP : 2.085
+  TYR : 3.035
+  VAL : 0.964136
```

Comment:
The "method_weights" line in the wts file I took this from has no explicit ordering. 

I have been comparing the per-residue energies with pyrosetta, and they don't seem to match these values. I had assumed they were alphabetical.

I've searched Rosetta for the code responsible for Ref and came up with https://github.com/RosettaCommons/main/blob/8e2068a4eb74cbb1a14335fab8254a5094600cae/source/src/core/energy_methods/ReferenceEnergy.cc#L76 (note the comment: "This is a terrible terrible terrible hack that will do for now.")

Is the ordering in the lookup table coming from https://github.com/RosettaCommons/main/blob/8e2068a4eb74cbb1a14335fab8254a5094600cae/source/src/core/chemical/AA.hh#L43 ?

Also, the function in ReferenceEnergy.cc looks quite a bit more complicated than a single lookup table. Are any of these other cases things we need to worry about right now?

@aleaverfay @fdimaio

Reply:
Presumably answered my own question by copying that ordering, as the score for ubq1 now matches rosetta.

So that just leaves the question of whether there's any more functionality from ReferenceEnergy.cc that may be missing.